core/less/buttons/buttons.less

```diff
@@ -143,26 +143,20 @@
 //
 // Styleguide 2.4
 
-.btn-primary {
-    &, &:hover, &:active, &:disabled, &[disabled], &.disabled {
-        color: white;
-        border-color: transparent;
-    }
-}
-.btn-secondary {
-    &, &:hover, &:active, &:disabled, &[disabled], &.disabled {
-        background-color: transparent;
-    }
-    &.selected {
```

Comment:
Ooops, yep, sorry. This must be the result of some hasteful merge. However, I must admit I was not quite happy with it. Especially the fact it had the color "white" hardwritten. Could we speak about it 5sec?